.github/workflows/e2e-agent.yml

```diff
@@ -59,11 +59,50 @@ jobs:
   global-environment:
     name: Import Global Environment
     uses: ./.github/workflows/required-reusable.yml
-
+  
+  build-e2e-image:
+    name: Build E2E Image
+    needs: global-environment
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 11
+      - uses: actions/cache@v4
+        with:
+          path: ~/.m2/repository
+          key: ${{ needs.global-environment.outputs.GLOBAL_CACHE_PREFIX }}-maven-third-party-e2e-cache-${{ github.sha }}
+          restore-keys: |
+            ${{ needs.global-environment.outputs.GLOBAL_CACHE_PREFIX }}-maven-third-party-e2e-cache-
+            ${{ needs.global-environment.outputs.GLOBAL_CACHE_PREFIX }}-maven-third-party-
+      - name: Build Project
+        run: ./mvnw -B clean install -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -DskipTests -Dspotless.apply.skip=true -Prelease
```

Comment:
`-Dmaven.javadoc.skip=true -Dcheckstyle.skip=true  -Dspotless.apply.skip=true` these parameters are not necessary as they are in check profile.

Reply:
Okay, it has been removed